### PR TITLE
[ticket/16260] Missing check whether the index exists in ACP - PHP 7.4

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16260

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16260

### CODE REVIEW

File: phpBB/includes/acp/acp_board.php [hunk 1]
```diff
@@ -530,11 +530,13 @@ function main($id, $mode)
 
 			if ($submit)
 			{
-				if (strpos($data['type'], 'password') === 0 && $config_value === '********')
+				if (strpos(isset($data['type']) ? $data['type'] : '', 'password') === 0 && $config_value === '********')
```
Comment (rubencm) [inline phpBB/includes/acp/acp_board.php]: if (strpos($data['type'] ?? '', 'password') === 0 && $config_value === '********')

is the same, but maybe more simple

Comment (mrgoldy) [inline phpBB/includes/acp/acp_board.php]: Could we not move the `isset()` outside of the `strpos()`?
As if it is not set, we don't have bother with rest anyways.

Comment (3D-I) [reply phpBB/includes/acp/acp_board.php]: Correct, the coalesce is simpler.

Comment (3D-I) [reply phpBB/includes/acp/acp_board.php]: Also doable yes. @marc1706 ?

Comment (marc1706) [reply phpBB/includes/acp/acp_board.php]: Yes, would make more sense.

**Comment by mrgoldy** [comment]
I was more thinking a long the lines of within the same if statement, rather than stacking them..
`if (isset($data['type']) && strpos($data['type'], 'password') === 0 && $config_value === '********')`

**Pull request merged**